Sources/MagicSDK/Modules/Auth/AuthModule.swift

```diff
@@ -35,6 +35,15 @@ public class AuthModule: BaseModule {
         }
     }
     
+    // MARK: - External Dismiss WebView
+    public func externalDismissLoginView() {
+        do {
+            try self.provider.overlay.detachWebViewFromCustomView()
```

Comment:
Would this call recycle webview?

Reply:
I considered potentially naming this method something like `authOnlyExternalDismissLoginView` 🤔 Perhaps add some more comments to inform web3 devs that they should not use this method.